trieste/bayesian_optimizer.py

```diff
@@ -500,6 +515,21 @@ def optimize(
                                 np.min(tagged_output[tag].observations),
                                 step=step,
                             )
+                        tf.summary.scalar(
+                            "Wallclock.total",
```

Comment:
Is "Wallclock" a standard phrase? Also I think our other scalars use lower case for the titles (unless they start with a class name or an objective):

```suggestion
                            "wallclock.total",
```

Reply:
@vpicheny is wallclock a standard phrase?